common/.eslintrc.fast.json

```diff
@@ -45,6 +57,82 @@
     ],
     "lines-between-class-members": ["error", "always", { "exceptAfterSingleLine": true }],
     "indent": "off",
-    "@typescript-eslint/indent": ["error", 2, { "ignoreComments": true, "flatTernaryExpressions": true }]
+    "@typescript-eslint/indent": ["error", 2, { "ignoreComments": true, "flatTernaryExpressions": true }],
+    "guard-for-in": "error",
+    "new-parens": "error",
+    "no-bitwise": "error",
+    "no-caller": "error",
+    "no-console": "error",
+    "no-eval": "error",
+    "no-new-wrappers": "error",
+    "no-shadow": "error",
+    "no-throw-literal": "error",
+    "no-trailing-spaces": "error",
+    "no-undef-init": "error",
+    "no-unused-expressions": "off",
+    "@typescript-eslint/no-unused-expressions": ["error"],
+    "no-var": "error",
+    "object-shorthand": "error",
+    "one-var": ["error", "never"],
+    "prefer-const": "error",
+    "prefer-arrow-callback": "error",
+    "quote-props": ["error", "as-needed"],
+    "radix": "error",
+    "spaced-comment": "error",
+    "eqeqeq": ["error", "smart"],
+    "quotes": "off",
+    "@typescript-eslint/quotes": ["error", "single", { "avoidEscape": true }],
+    "curly": ["error", "multi-line"],
+    "comma-dangle": ["error", {
+        "arrays": "always-multiline",
+        "objects": "always-multiline",
+        "imports": "always-multiline",
+        "exports": "always-multiline",
+        "functions": "only-multiline"
+    }],
+    "no-whitespace-before-property": "error",
+    "space-infix-ops": "error",
+    "keyword-spacing": "error",
+    "key-spacing": "error",
+    "space-before-blocks": "error",
+    "space-before-function-paren": "off",
+    "@typescript-eslint/space-before-function-paren": ["error", {
+        "anonymous": "never",
+        "named": "never",
+        "asyncArrow": "always"
+    }],
+    "space-in-parens": "error",
+    "space-unary-ops": "error",
```

Comment:
This makes sense (only) because of the default value.  Maybe specify it explicitly or in a comment (assuming this is JSON-with-comments), to be clear that this line is actually a _good_ thing?